cli/root.go

```diff
@@ -461,6 +461,12 @@ func (r *RootCmd) Command(subcommands []*serpent.Command) (*serpent.Command, err
 			Value:       serpent.StringOf(&r.globalConfig),
 			Group:       globalGroup,
 		},
+		{
+			Flag:        cliui.NoColorFlag,
```

Comment:
Wonder if it would make sense to also allow setting it as an env (`CODER_NO_COLOR`)? Not blocking.

Reply:
Yes that should be a good idea. There is a 'standard' around using `NO_COLOR` for this https://no-color.org. We could do both `NO_COLOR` and `CODER_NO_COLOR`?

Reply:
Sounds good to me!